### PR TITLE
[Merged by Bors] - feat: deprecate`fluvio install`

### PR DESCRIPTION
**Pull request by EstebanBorai**
Deprecates `fluvio install` to use `fvm install` instead.

### CODE REVIEW

File: crates/fluvio-cli/src/install/plugins.rs [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+    /// override default target arch determination
+    #[arg(long, hide_short_help = true)]
+    pub target: Option<String>,
+}
+
+impl InstallOpt {
+    pub async fn process(self) -> Result<()> {
+        println!("warning: `fluvio install` is deprecated, use `fvm install` instead.");
+        println!("Refer to https://www.fluvio.io/docs/get-started/linux/#install-fluvio-cli");
```
Comment (EstebanBorai) [inline crates/fluvio-cli/src/install/plugins.rs:56]: Adds deprecation notice and link to install docs.

**Comment by EstebanBorai** [comment]
@tjtelan I added a deprecation notice on `fluvio install` as we move into `fvm install` instead.

Do you have any recommendations on a documentation link to point to as part of the warning message?

As we release this on `0.11.0` users could visit FVM section in the docs to read more.

**Review by digikata — APPROVED** [review]
LGTM

**Pull request closed without merging**